### ISSUE TITLE
Primefaces.bcnu doesn't pass parameters correctly

### ISSUE THREAD

**Issue by asw12** [primefaces/primefaces#721]
As of the commit [ee9e2f2b68cac7d576e86fbb52ac1798f548d66c](https://github.com/primefaces/primefaces/commit/ee9e2f2b68cac7d576e86fbb52ac1798f548d66c), it appears as though multiple ClientBehaviors on the same event for an input should be supported as long as they are non obstrusive.

However, the [arguments list passed to the chained javascript callbacks](https://github.com/primefaces/primefaces/blob/bb96e7bb40924077ec587d70f7e5c2cfa5de7cf1/src/main/resources/META-INF/resources/primefaces/core/core.js#L511) is not correct in the "bcnu" method. The first argument to [fn.call](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) in javascript is the "this" argument, so the form parameters object in "ext" is being set to "this" for the ajax binding function call, rather than being passed in as arg1.

Here is a reproduction example that I tested in PrimeFaces 5.2. The first autocomplete throws a NullPointerException when an item is selected.

```
<h:form>
    <p:growl id="msgs" showDetail="true" />

    <p:autoComplete id="ac1" value="#{autoCompleteView.multiAjaxValue}"
                    completeMethod="#{autoCompleteView.completeText}">
        <p:ajax event="itemSelect" listener="#{autoCompleteView.onItemSelect1}" update="msgs"/>
        <!-- Really obtuse example where the second ajax event is almost identical to the first -->
        <p:ajax event="itemSelect" listener="#{autoCompleteView.onItemSelect2}" update="msgs"/>
    </p:autoComplete>
    <p:autoComplete id="ac2" value="#{autoCompleteView.singleAjaxValue}"
                    completeMethod="#{autoCompleteView.completeText}">
        <p:ajax event="itemSelect" listener="#{autoCompleteView.onItemSelect1}" update="msgs"/>
    </p:autoComplete>
</h:form>



@ViewScoped
@Named("autoCompleteView")
public class AutoCompleteView implements Serializable {
    private String multiAjaxValue;
    private String singleAjaxValue;

    public List<String> completeText(String query) {
        List<String> results = new ArrayList<>();
        for(int i = 0; i < 10; i++) {
            results.add(query + i);
        }

        return results;
    }

    public void onItemSelect1(SelectEvent event) {
        FacesContext.getCurrentInstance().addMessage(null, new FacesMessage("Item Selected", event.getObject().toString()));
    }

    public void onItemSelect2(SelectEvent event) {
        FacesContext.getCurrentInstance().addMessage(null, new FacesMessage("Item Selected", event.getObject().toString()));
    }

    public String getMultiAjaxValue() {
        return multiAjaxValue;
    }

    public void setMultiAjaxValue(String multiAjaxValue) {
        this.multiAjaxValue = multiAjaxValue;
    }

    public String getSingleAjaxValue() {
        return singleAjaxValue;
    }

    public void setSingleAjaxValue(String singleAjaxValue) {
        this.singleAjaxValue = singleAjaxValue;
    }
}
```

**Comment by cghislai** [comment]
I confirm this happen on Primefaces 5.2 and 5.3 while it was working on Primefaces 5.1.
This prevents composite component to expose client behaviors if they already handle them internally.

As a workaround, one could create an hidden commandButton, triggers a 'click' on the first ajax call and make the second one listen for the click event:

Using the code below, if users of the composite component register an ajax listener for the change event, then form parameters will not be correctly encoded.

```
<cc:interface>
  <cc:clientBehavior name="change" targets="myCompA" event="dateSelect"/>
</cc:interface>
<cc:implementation>
  <p:schedule id="myCompA">
   <p:ajax event="dateSelect" listener="#{myBean.action()}"/>
</p:schedule>
</cc:implemtation>
```

Working workaround:

```
<cc:interface>
  <cc:clientBehavior name="change" targets="myWorkaroundButton" event="click"/>
</cc:interface>
<cc:implementation>
<p:commandButton id="myWorkaroundButton" widgetVar="myWorkaroundButton"
style="display: none;"/>
  <p:schedule id="myCompA">
   <p:ajax event="dateSelect" listener="#{myBean.action()}"
     oncomplete="$(PF('myWorkaroundButton').jqId).trigger('click');" />
</p:schedule>
</cc:implemtation>
```

**Comment by tandraschko** [comment]
Would be great if you could provide a PR.

**Comment by cghislai** [comment]
this does not seem to be fixed in primefaces 6.2
